tremor-script/src/ast/raw.rs

```diff
@@ -1037,10 +1037,26 @@ impl<'script> Upable<'script> for ImutExprRaw<'script> {
                             let i = e.up(helper)?;
                             let i = i.try_reduce(helper)?;
                             match i {
+                                #[cfg(not(feature = "erlang-float-testing"))]
                                 ImutExprInt::Literal(l) => l.value.as_str().map_or_else(
                                     || StrLitElement::Lit(l.value.encode().into()),
                                     |s| StrLitElement::Lit(s.to_string().into()),
                                 ),
+                                // TODO: The float scenario is different in erlang and rust
+                                // We knowingly excluded float correctness in string interpolation
+                                // as we don't want to over engineer and write own format functions.
+                                // any suggestions are welcome
+                                #[cfg(feature = "erlang-float-testing")]
```

Comment:
Tarpaulin is mean here we got to exclude this code from coverage

```suggestion
                                #[cfg(feature = "erlang-float-testing")]
                                #[cfg(not(tarpaulin_include))]
```

Reply:
I will add the same in immutable expression (imut_expr.rs) as well.